src/extension.js

```diff
@@ -0,0 +1,89 @@
+import glob from 'glob';
+import path from 'path';
+import resolve from 'resolve';
+import yeoman from 'yeoman-environment';
+
+/**
+ * Class representing an Extension
+* */
+class Extension {
+  /**
+   * When an extension is instantiated, pass in the object in which to register
+   * with and the module keyword; Run discoverChildren to register commands and
+   * generators with Stanza.
+   *
+   * @name constructor
```

Comment:
ESDoc should pick this stuff up automatically. @memberof isn't even a supported tag. See: https://esdoc.org/tags.html and https://esdoc.org/tutorial.html.